### PR TITLE
Update Client docs for Shared Memory

### PR DESCRIPTION
**Pull request by CoderHam**
Document the shared memory features as a Beta release

### CODE REVIEW

File: docs/client.rst [hunk 1]
```diff
@@ -34,7 +34,8 @@ The inference server *client libraries* make it easy to communicate
 with the TensorRT Inference Server from your C++ or Python
 application. Using these libraries you can send either HTTP or GRPC
 requests to the server to check status or health and to make inference
-requests.
+requests. These libraries support using Shared Memory for passing
```
Comment (deadeyegoodwin) [inline docs/client.rst]: "...libraries also support..."

File: docs/client.rst [hunk 2]
```diff
@@ -98,8 +99,13 @@ configured with all the dependencies required to run those examples
 within the container. The easiest way to try the examples described in
 the following sections is to run the client image with -\\-net=host so
 that the client examples can access the inference server running in
-its own container (see :ref:`section-running-the-inference-server` for
-more information about running the inference server)::
+its own container. To be able to use shared memory you need to run the
+client and server image with -\\-ipc=host so that the inference server
+can access the shared memory in the client container. Additionally, to
+create shared memory regions that are larger than 64MB, the
+-\\-shm-size=1g flag is needed while running the client image.
```
Comment (deadeyegoodwin) [inline docs/client.rst]: Remove period

**Comment by deadeyegoodwin** [comment]
@CoderHam Only minor changes were needed and I need to do doc updates anyway... so I will do them.

**Comment by CoderHam** [comment]
Thanks @deadeyegoodwin

**Pull request merged**